net.hachune.mikutter.yaml

```diff
@@ -0,0 +1,69 @@
+app-id: net.hachune.mikutter
```

Comment:
Please reformat it according to our [style guide](https://github.com/flathub/flathub/wiki/YAML-Style-Guide).